src/drive/web/modules/filelist/AddFolder.jsx

```diff
@@ -58,6 +59,19 @@ const AddFolder = ({
           >
             —
           </TableCell>
+          {!isMobile &&
```

Comment:
I think we should not have the isMObile condition here, and just base ourselves on the presence of additionalColumns.